src/share/classes/com/sun/javatest/regtest/agent/JUnitRunner.java

```diff
@@ -114,60 +125,154 @@ private static void runWithJUnitPlatform(Class<?> mainClass) throws Exception {
         Thread.currentThread().setContextClassLoader(mainClass.getClassLoader());
         try {
             String testQuery = System.getProperty("test.query");
-            // if test.query is set, treat it as a method name to be executed
             LauncherDiscoveryRequest request = LauncherDiscoveryRequestBuilder.request()
+                // if test.query is set, treat it as a method name to be executed
```

Comment:
Note sure why you have moved this into the middle of chained function calls. If anything, I would move it to before line 127, which is the line that actually uses `test.query`, as compared to `testQuery`